src/core/panel/Panel.ts

```diff
@@ -321,7 +321,13 @@ class Panel {
     if (cached) {
       this._size = cached.size;
       this._margin = { ...cached.margin };
-      this._height = cached.height;
+      this._height = horizontal ? getElementSize({
+        el,
+        horizontal: false,
+        useFractionalSize,
+        useOffset: true,
+        style: getStyle(el)
+      }) : cached.height;
```

Comment:
This affects all other cases not using `panelsPerView`, this should be reverted.
Or, you should check whether to calculate the height by `cached.height` not `horizontal`

Reply:
`horiztontal` I meant on the previous comment was to update height only when using `panelsPerView` and `horizontal: true` as when `horitzontal: false` all panels will use all same heights.

Reply:
Currently it seems `cached` is passed to `resize` only via `updatePanelSizeByGrid` when `panelsPerView` is greater than 0.
However considering the purpose of `resize`, it seems better to determine whether to include `height` in `firstPanelSizeObj` in `updatePanelSizeByGrid`.